tools/dotnet-msidentity/README.md

```diff
@@ -199,6 +219,6 @@ The tool supports ASP.NET Core applications created with .NET 5.0 and netcoreapp
 
 ```Shell
 dotnet new blazorwasm --auth SingleOrg --framework netcoreapp3.1
-dotnet-msidentity
+dotnet msidentity
 dotnet run -f netstandard2.1
```

Comment:
It looks like on line `221` we tell them to create the project with `--framework netcoreapp3.1` and the on line `222` we tell them to run with 2.1 `dotnet run -f netstandard2.1`. That doesn't seem right.